### PR TITLE
[11.0] [mig] stock_picking_report_custom_description

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: stock_picking_report_custom_description/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+#. Create a sale order and set manual descriprion in sale order lines.
```
Comment (pedrobaeza) [inline stock_picking_report_custom_description/readme/USAGE.rst]: s/descriprion/description

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Remove from root `__init__.py` the import of models folder.

I'm missing the view in backend UI of the `name` field in both "Operations" and "Detailed operations".

**Comment by pedrobaeza** [comment]
You can also squash a bit commit history

**Review by tafaRU — CHANGES_REQUESTED** [review]
@sergio-teruel please note that Travis failed due to https://travis-ci.org/OCA/stock-logistics-reporting/jobs/433041027#L1285 beacause of https://github.com/Tecnativa/stock-logistics-reporting/blob/11.0-mig-stock_picking_report_custom_description/stock_picking_report_custom_description/__init__.py#L2

File: stock_picking_report_custom_description/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Picking Report Custom Description",
+    "summary": "Show moves description in picking reports",
+    "version": "11.0.1.0.0",
+    "category": "Warehouse",
+    "website": "https://www.tecnativa.com",
```
Comment (bodedra) [inline stock_picking_report_custom_description/__manifest__.py]: Shouldn't be https://github.com/OCA/stock-logistics-reporting ?

File: stock_picking_report_custom_description/views/report_deliveryslip.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (bodedra) [inline stock_picking_report_custom_description/views/report_deliveryslip.xml:1]: Please remove `encoding="UTF-8"`

File: stock_picking_report_custom_description/views/report_stockpicking_operations.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bodedra) [inline stock_picking_report_custom_description/views/report_stockpicking_operations.xml:1]: Same here. Remove `encoding="UTF-8"`

File: stock_picking_report_custom_description/views/stock_report.xml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (bodedra) [inline stock_picking_report_custom_description/views/stock_report.xml:1]: Same as above

**Comment by pedrobaeza** [comment]
@sergio-teruel can you please finish this?

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@tafaRU @bodedra  everything attended.

**Review by bodedra — APPROVED** [review]
Code Review :+1:

**Review by tafaRU — APPROVED** [review]
Both functional and technical :+1:

**Pull request merged**